### ISSUE TITLE
Is this library unmantained?

### ISSUE THREAD

**Issue by drenda** [telegram-s/telegram-api-old#52]
Hi,
is this library dead? i made a try to make it works but I receive a timeout during registration.

Thanks

**Comment by yasserzamani** [comment]
I don't know if library is dead but recently I tried that and changing ip to [these ips](https://github.com/telegramdesktop/tdesktop/blob/master/Telegram/SourceFiles/config.h#L221) resolved timeout.

**Comment by yasserzamani** [comment]
Yes, this library is very old according to [this so question - Telegram Api layer 52, super groups and channels](http://stackoverflow.com/questions/38801700/telegram-api-layer-52-super-groups-and-channels) which says:

> i am java developer . i found telegram java api in github .but this api is old . i want to make new library for java (support channels and superGroups)
